Documentation/cmdref/cilium-agent_cmdref.md

```diff
@@ -0,0 +1,20 @@
+<!-- This file was autogenerated via cilium-agent --cmdref, do not edit manually-->
```

Comment:
The generated files mention `cilium-agent --cmdref`, but with the rest of the commit I would expect `cilium-agent cmdref`? Looks like this comes from `cilium-health/cmd/root.go`?

Reply:
You mean `daemon/cmd/cmdref.go`? Yep looks like I forgot to update that. Will fix.

Reply:
Ah right, the comment `cilium-health/cmd/root.go` seems wrong but looks like a separate error :) Thanks for updating!